pkg/modules/AWS/eks/locals.tf

```diff
@@ -0,0 +1,4 @@
+
+locals {
+  cluster_asg_name = try(module.eks.node_groups.appNodes.resources[0].autoscaling_groups[0].name, null)
```

Comment:
Can we add a comment on why we have `try` here?

Reply:
done